### ISSUE TITLE
[bug]: Wrong saving when Ctrl + S and Edit request's directly

### ISSUE THREAD

**Issue by bachdgvn** [hoppscotch/hoppscotch#2297]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Wrong saving when Ctrl + S and Edit request's directly.

### Steps to reproduce

Step 1: Duplicate a request in folder
Step 2: Rename it in Collections view
Step 3: Change URL in Main view
Step 4: Ctrl + S

Now: request's name is old name, not edited name.
Expected: request's name is edited name.

### Environment

Production

### Version

Cloud

**Comment by TeamPIDU** [comment]
Yes facing the same issue, the Name of the request is getting changed to the old name as soon as we Save changes in Request Window - please look into this as this is making work very hard - have to refresh the whole thing everytime this happens to get it working.
